collector/src/prometheus.h

```diff
@@ -0,0 +1,49 @@
+/*  promethus.h
+ *
+ *  This library aims to enable Metrics-Driven Development for C++ services.
```

Comment:
align this top-of-file comment to other files, i.e. put here the GPLv3 license... of course if you agree in releasing your code under GPLv3 :)